index.html

```diff
@@ -38,7 +38,7 @@
   </head>
   <body>
     <header>
-      <h1>My Name</h1>
+      <h1>Haythem Mohammed</h1>
```

Comment:
This is a good start, make sure to replace the readme and up your seo score to 100 on lighthouse